Documentation/reportgenerationqueries.md

```diff
@@ -164,6 +165,18 @@ spec:
     ORDER BY pod_request_memory_byte_seconds DESC
 ```
 
+## Modifying Columns For Report Display
+
+You can modify the custom resource to display all columns or to hide or show columns as needed. The full endpoint displays all columns.
+An example report for `namespace-cpu-request` in JSON for the full endpoint would be:
```

Comment:
This isn't an example report, it's an example of querying a report results from the reporting-operator API. Can you rephrase this to be more accurate?